### DISCUSSION TITLE
ghas-cli - CLI utility to deploy at scale and interact with GitHub Advanced Security

### DISCUSSION THREAD

**Discussion by jboursier-mwb** [NickLiffen/ghas-enablement#93] [Show and tell]
Hello,

I just wanted to share a repository I've been working on and using to deploy GHAS at Malwarebytes: https://github.com/Malwarebytes/ghas-cli .

I initially wanted to use `ghas-enablement` to deploy GHAS on all repositories, but I faced too many restrictions due to our environment (see "Why not use ghas-enablement?" on the readme: https://github.com/Malwarebytes/ghas-cli#why-not-use-ghas-enablement) so I decided to take a stab and implement a cli utility to deploy and interact with GHAS for a large organization.

It's open-source and public, and welcome for contributions and feedbacks!

Hopefully some of you can find it useful, and I hope it can eventually provide some improvement ideas to this repository!

**Comment by NickLiffen** [comment]
Hey @jboursier-mwb 👋 

**Thank you** for putting this together ❤️ 

I love what you have done 🙇 

I am wondering if you think we can add any of [these problems](https://github.com/Malwarebytes/ghas-cli#why-not-use-ghas-enablement) to this tool to make it more usable 👍 

I love what you have done, and I think the problems you have identified should be addressed by this tool 👍 I think it would be great to create issues for all of these and get them addressed in this tool. Would that be something you are 👍 with? I can do the work 👍 

Great tool you have built though ❤️
